### ISSUE TITLE
Failed to minify

### ISSUE THREAD

**Issue by kruyvanna** [JedWatson/react-select#1978]
When i run build on my app, it says:

    Failed to minify the code from this file: 

 	./node_modules/react-select/lib/Select.js:39 

    Read more here: http://bit.ly/2tRViJ9

**Comment by o33ik** [comment]
I have a similar problem
`remote: Errors prevented bundling:
remote: While minifying app code:
remote: packages/minifyStdJS/plugin/minify-js.js:96:26: Babili minification error
remote: within packages/modules.js:
remote: node_modules/react-select/lib/Select.js, line 1167
remote:
remote: Only one default export allowed per module.:
remote:
remote: export default Select;
remote:
remote: at maybeThrowMinifyErrorBySourceFile`

**Comment by o33ik** [comment]
I have a similar problem
`
remote: Errors prevented bundling:
remote: While minifying app code:
remote: packages/minifyStdJS/plugin/minify-js.js:96:26: Babili minification error
remote: within packages/modules.js:
remote: node_modules/react-select/lib/Select.js, line 1167
remote:
remote: Only one default export allowed per module.:
remote:
remote: export default Select;
remote:
remote: at maybeThrowMinifyErrorBySourceFile
`

**Comment by arvinsim** [comment]
I suddenly have the same problem today

```
`static/js/main.3d6a0c42.js from UglifyJs
Unexpected token: name (instanceId) [./~/react-select/lib/Select.js:39,0][static/js/main.3d6a0c42.js:19603,4]`

I am on `react-select=1.0.0-rc.5`

**Comment by o33ik** [comment]
updated package to 1.0.0-rc.7 and problem is gone

**Comment by gwyneplaine** [comment]
Hi, @kruyvanna this should be fixed in 1.0.0-rc.7, please let us know if this is not the case

**Comment by kruyvanna** [comment]
@gwyneplaine I confirm fixed in 1.0.0-rc.7
Thanks
